ckanapi/meta.yaml

```diff
@@ -0,0 +1,74 @@
+package:
+  name: ckanapi
+  version: "3.5"
+
+source:
+  fn: ckanapi-3.5.tar.gz
+  url: https://pypi.python.org/packages/source/c/ckanapi/ckanapi-3.5.tar.gz
+  md5: 2ab2508df730b0e7d84e13be74adc314
+#  patches:
+   # List any patch files here
+   # - fix.patch
+
+build:
+  preserve_egg_dir: True
+  entry_points:
+    # Put any entry points (scripts to be generated automatically) here. The
+    # syntax is module:function.  For example
+    #
+    # - ckanapi = ckanapi:main
+    #
+    # Would create an entry point called ckanapi that calls ckanapi.main()
+
+    - ckanapi=ckanapi.cli.main:main
+
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  # number: 1
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - setuptools
+    - docopt
+    - requests
+    - simplejson
```

Comment:
If you add `setuptools` we can add `--single-version-externally-managed --record record.txt` you can probably remove `docopt`, `requests`, and `simplejson` from the build step.